pkg/performanceprofile/controller/utils.go

```diff
@@ -0,0 +1,51 @@
+package controller
```

Comment:
can we avoid `utils.go` perhaps? (semi-related: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common)

Reply:
I totally agree that fuzzy package names can lead to a jumble package with different functions but I didn't came with a better name for this and as `util` package is commonly used in our projects took that path.

I enclosed the functionality in a function to be able to test it easely ( not need to restart operator to execute the sut code )